.eslintrc.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
why you do this ?

Reply:
otherwise it complains about stuff in the json module since they dont get used by anything now.